pcsx2/Recording/RecordingInputManager.cpp

```diff
@@ -32,10 +32,6 @@ RecordingInputManager::RecordingInputManager()
 
 void RecordingInputManager::ControllerInterrupt(u8 & data, u8 & port, u16 & BufCount, u8 buf[])
 {
-	if (port >= 2)
-	{
-		return;
-	}
```

Comment:
leave alone until multi-tap?

Reply:
The problem here is that sio.port will only ever be 0 or 1. It's when sio.slot[sio.port] is used that the values can be greater, but nothing was actually pulling that. Everything that is called in sio.cpp only ever used sio.port as the parameter, so having these here made no difference.
Addtitionally, I added a slot check in sio.cpp that makes sure to only call those functions only if it's writing controllers 1A & 2A. (https://github.com/PCSX2/pcsx2/pull/3669/commits/7ed21d4c5ae1a696b6c40f38636e43a406964bdd)

Reply:
Right, I saw that near the end of the review.  All of these checks are redundant given that top-level check, good cleanup!  Though I think in the future we'll want to shift that check back into the input recording's code, just for no reason other than reducing the recording footprint in core's code.  But a later concern as it will soon change with your multitap PR